Backend/app.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const app = express();
+// manage db
+const models = require('./models');
+// routers
+const router = express.Router();
+const rootrouter = require('./router/index');
+
+// to get json data from body
+const bodyParser = require('body-parser');
+
+models.sequelize.sync();
+
+//json encoded
+app.use(bodyParser.json());
+//url encoded
+app.use(bodyParser.urlencoded({
+  extended: true,
+}));
+
+app.use('/api', rootrouter);
+
+app.listen(3000, function(){
```

Comment:
@eunyyyoung why 3000?

Reply:
There is no reason.  Just in order to run in localhost.... @mtnviewlab

Reply:
@eunyyyoung does front end team knows about the port 3000?
Make sure they know it, so that they can add port number at the end of domain name.
The well known ports for http is 80.

Reply:
@mtnviewlab thank you.